src/Abstractions/NexusMods.Abstractions.Games/ASortableItemProviderFactory.cs

```diff
@@ -0,0 +1,143 @@
+using System.ComponentModel;
+using System.Diagnostics;
+using DynamicData;
+using NexusMods.Abstractions.Loadouts;
+using NexusMods.Abstractions.NexusWebApi.Types.V2;
+using NexusMods.MnemonicDB.Abstractions;
+using R3;
+
+namespace NexusMods.Abstractions.Games;
+
+/// <inheritdoc />
+public abstract class ASortableItemProviderFactory : ISortableItemProviderFactory
+{
+    private readonly IConnection _connection;
+    private readonly Dictionary<LoadoutId, ILoadoutSortableItemProvider> _providers = new();
+    
+    /// <inheritdoc />
+    public abstract Guid SortOrderTypeId { get; }
+    
+    /// <inheritdoc />
+    public abstract string LearnMoreUrl { get; }
+    
+    /// <inheritdoc />
+    public virtual string SortOrderName => "Load Order";
+    
+    /// <inheritdoc />
+    public virtual string OverrideInfoMessage => string.Empty;
+
+    /// <inheritdoc />
+    public virtual string OverrideInfoTitle => string.Empty;
+    
+    /// <inheritdoc />
+    public virtual string WinnerIndexToolTip => "Last Loaded Mod Wins: Items that load last will overwrite changes from items loaded before them.";
+
+    /// <inheritdoc />
+    public virtual string IndexColumnHeader => "LOAD ORDER";
+
+    /// <inheritdoc />
+    public virtual string DisplayNameColumnHeader => "NAME";
+
+    /// <inheritdoc />
+    public virtual string EmptyStateMessageTitle => "No Sortable Mods detected";
+
+    /// <inheritdoc />
+    public virtual string EmptyStateMessageContents => "Some mods may modify the same game assets. When detected, they will be sortable via this interface.";
```

Comment:
Can we group these UI messages into a type? `LoadOrderUiMessages` or something. It seems odd to have this in here.

Reply:
Done